aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb

```diff
@@ -0,0 +1,239 @@
+    "In this step, we will build SageMaker endpoint from scratch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d955679",
+   "metadata": {},
+   "source": [
+    "### Getting the container image URI\n",
+    "\n",
+    "[Large Model Inference available DLC](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a174b36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_uri = image_uris.retrieve(\n",
+    "        framework=\"djl-tensorrtllm\",\n",
+    "        region=sess.boto_session.region_name,\n",
+    "        version=\"0.25.0\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11601839",
+   "metadata": {},
+   "source": [
+    "### Upload artifact on S3 and create SageMaker model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "38b1e5ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s3_code_prefix = \"large-model-lmi/code\"\n",
+    "bucket = sess.default_bucket()  # bucket to house artifacts\n",
+    "code_artifact = sess.upload_data(\"mymodel.tar.gz\", bucket, s3_code_prefix)\n",
+    "print(f\"S3 Code or Model tar ball uploaded to --- > {code_artifact}\")\n",
+    "\n",
+    "model = Model(image_uri=image_uri, model_data=code_artifact, role=role)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "004f39f6",
+   "metadata": {},
+   "source": [
+    "### 4.2 Create SageMaker endpoint\n",
+    "\n",
+    "You need to specify the instance to use and endpoint names"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8e0e61cd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "instance_type = \"ml.g5.12xlarge\"\n",
+    "endpoint_name = sagemaker.utils.name_from_base(\"lmi-model\")\n",
+    "\n",
+    "model.deploy(initial_instance_count=1,\n",
+    "             instance_type=instance_type,\n",
+    "             endpoint_name=endpoint_name,\n",
+    "             # container_startup_health_check_timeout=3600\n",
+    "            )\n",
+    "\n",
+    "# our requests and responses will be in json format so we specify the serializer and the deserializer\n",
+    "predictor = sagemaker.Predictor(\n",
+    "    endpoint_name=endpoint_name,\n",
+    "    sagemaker_session=sess,\n",
+    "    serializer=serializers.JSONSerializer(),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb63ee65",
+   "metadata": {},
+   "source": [
+    "## Step 5: Test and benchmark the inference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2bcef095",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictor.predict(\n",
+    "    {\"inputs\": \"tell me a story of the little red riding hood\", \"parameters\": {\"max_new_tokens\":128, \"do_sample\":True}}\n",
```

Comment:
"True" is not a valid json constant, it should be lower case

Reply:
I only changed true to True so it can be used when running the jupyter notebook (there it's a python dictionary and not a json object)